src/components/dialogmote/motehistorikk/MotehistorikkPanel.tsx

```diff
@@ -120,9 +120,6 @@ export const MotehistorikkPanel = ({
   dialogmoteunntak,
   historiskeMoter,
 }: MotehistorikkPanelProps) => {
-  if (historiskeMoter.length === 0 && dialogmoteunntak.length === 0)
```

Comment:
Burde det vært `return null` her ? Blir det ikke nå bare et tomt panel med header og subtitle.

Reply:
Litt vanskelig PR å lese😬 Denne er flyttet ut til der den brukes i `Motelandingsside` for å gjør det litt mer tydelig når en komponent vises eller ikke fra der den renderes.
@JMLindseth løftet det på en [tidligere PR](https://github.com/navikt/syfomodiaperson/pull/1068#issuecomment-1798712904)